app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentPresenter.kt

```diff
@@ -485,7 +515,11 @@ class AdyenPaymentPresenter(private val view: AdyenPaymentView,
   private fun handleErrors(error: Error) {
     when {
       error.isNetworkError -> view.showNetworkError()
-      error.code != null -> view.showSpecificError(servicesErrorCodeMapper.mapError(error.code!!))
+      error.code != null -> {
+        val resId = servicesErrorCodeMapper.mapError(error.code!!)
+        if (error.code == 403) handleFraudFlow(resId)
```

Comment:
Add 403 as static variable HTTP_FRAUD, or maybe add an util with the different http meanings to use in the feature (right now it would only have 403 and 409 I think)